sw_stm32/Drivers/CMSIS/Include/cmsis_gcc.h

```diff
@@ -793,7 +793,7 @@ __STATIC_FORCEINLINE void __set_FPSCR(uint32_t fpscr)
 {
 #if ((defined (__FPU_PRESENT) && (__FPU_PRESENT == 1U)) && \
      (defined (__FPU_USED   ) && (__FPU_USED    == 1U))     )
-#if __has_builtin(__builtin_arm_set_fpscr)
+#if 1 || __has_builtin(__builtin_arm_set_fpscr)
```

Comment:
Is this a quick patch?  If yes is it still needed or can it be removed again.   If not please add a comment to describe why this is needed.